openlibrary/macros/ShareModal.html

```diff
@@ -49,6 +49,20 @@
               </a>
               <div class="share-source">$_('Embed')</div>
             </li>
+
+            <li>
+              <a class="copy-url-btn" title="$_('Copy url to clipboard')" data-ol-link-track="Share|CopyURL"
+                  onclick="navigator.clipboard.writeText('openlibrary.org'+'$page_url');">
+                  <img
+                      alt="CopyURL icon"
```

Comment:
```suggestion
                      alt="$_('Copy URL icon')"
```
Add space between `Copy` and `URL`.  Use localized strings (more information about this [here](https://github.com/internetarchive/openlibrary/wiki/Internationalization-Contributor's-Guide-(i18n)#internationalization-i18n-developers-guide)).